R/pkg/R/DataFrame.R

```diff
@@ -2649,15 +2651,15 @@ setMethod("merge",
 #' @return list of columns
 #'
 #' @note generateAliasesForIntersectedCols since 1.6.0
-generateAliasesForIntersectedCols <- function (x, intersectedColNames, suffix) {
+generateAliasesForIntersectedCols <- function(x, intersectedColNames, suffix) { # nolint
```

Comment:
Looks this hits 30 length identifier limit but we can't change as exposed in the doc.

Reply:
wait... this isn't public though; it's not in https://github.com/apache/spark/blob/master/R/pkg/NAMESPACE
looks like this shouldn't be in the doc (`@noRd` or `#` instead of `#'`)